### PR TITLE
v4.1.1

### PR DESCRIPTION
**Pull request by studioswong**
## Description

Patch fix release for experiment tracking

## Development notes

I have updated our release notes and version

## QA notes

N/A

## Checklist

- [x] Read the [contributing](/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added new entries to the `RELEASE.md` file
- [ ] Added tests to cover my changes

### CODE REVIEW

**Review by rashidakanchwala — APPROVED** [review]
lgtm!

File: RELEASE.md [hunk 1]
```diff
@@ -7,6 +7,13 @@ Please follow the established format:
 - Use present tense (e.g. 'Add new feature')
 - Include the ID number for the related PR (or PRs) in parentheses
 -->
+
+# Release 4.1.1
+## Bug fixes and other changes
+
+- Fix display of boolean values on tracking dataset (#652)
+- Fix node IDs in test data after Kedro updates its _str_logic for node (#653)
```
Comment (limdauto) [inline RELEASE.md:15]: As discussed yesterday, I don't think this needs to go into the release note. It's not userfacing.

**Review by limdauto — APPROVED** [review]
LGTM!

File: RELEASE.md (hunk 1 above)
Comment (studioswong) [reply RELEASE.md:15]: I see- not sure if we shouldn't include non-user facing notes given that we are open source, moreover us having the practice of including fixes and changes to the server on our previous release notes?

**Pull request merged**